ads/google/imaVideo.js

```diff
@@ -221,6 +223,35 @@ let consentState;
 // Throttle for the showControls() function
 let showControlsThrottled = throttle(window, showControls, 1000);
 
+/**
+ * @param {!Document} document
+ * @param {?Array<Array<string|Object>>} childrenDef
+ *   an array of [tagName, attributes] items like:
+ *     [
+ *       ['SOURCE', {'src': 'foo.mp4'}],
+ *       ['TRACK', {'src': 'bar.mp4'}],
+ *     ]
```

Comment:
FMI: Why is a 2-item array preferred to an object with named properties?

Reply:
No good reason, I just like that it looks like the signatures for `Preact.createElement` and our `createElementWithAttributes`. Iframe names also have a size limit but doubt we would reach it with `<source>`s anyway.